apps/vmq_diversity/src/vmq_diversity_mysql.erl

```diff
@@ -124,6 +124,7 @@ hash_method(_, St) ->
         password -> <<"PASSWORD(?)">>;
         md5 -> <<"MD5(?)">>;
         sha1 -> <<"SHA1(?)">>;
-        sha256 -> <<"SHA2(?, 256)">>
-    end, 
+        sha256 -> <<"SHA2(?, 256)">>;
+        _ -> HashMethod
```

Comment:
@larshesel Can you tell if it's `HashMethod` or `<<HashMethod>>` ? I ain't exactly fluid in Erlang.

Reply:
`HashMethod` would be correct. But I don't think you'd need to change the `password`, `md5`, `sha1` and `sha256` to actual strings instead of atoms.

Reply:
@dergraf Excuse me. Did you mean the keys **do** or **do not** need to be quoted?

Reply:
I do agree, atoms should be kept as an enum field, having the user enter free text with no validation for this setting could result in errors and lots of support requests,